### PR TITLE
update the pull request template

### PR DESCRIPTION
**Pull request by xabbuh**
| Q             | A
| ------------- | ---
| Branch?       | 2.7
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | 
| License       | MIT
| Doc PR        |

### CODE REVIEW

**Review by fabpot — APPROVED** [review]

**Review by Simperfit — APPROVED** [review]

**Comment by chalasr** [comment]
Thanks @xabbuh.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 | Q             | A
 | ------------- | ---
-| Branch?       | 3.4 or master / 2.7, 2.8 or 3.3 <!-- see comment below -->
+| Branch?       | master / 2.7, 2.8, 3.3, or 3.4 <!-- see comment below -->
```
Comment (nicolas-grekas) [inline .github/PULL_REQUEST_TEMPLATE.md]: this is going to need another change in 4 weeks to add 4.0, better wait?

Comment (nicolas-grekas) [reply .github/PULL_REQUEST_TEMPLATE.md]: race condition :)

Comment (chalasr) [reply .github/PULL_REQUEST_TEMPLATE.md]: I was thinking the same, but now that the PR is done...

Comment (fabpot) [reply .github/PULL_REQUEST_TEMPLATE.md]: I think it's good to have merged it now. Another in 4 weeks, no big deal :)

**Pull request merged**